[WIP] start:dev script with HMR and lint using local librairies (#3291)
gridsuite/gridstudy-app: README.md

If you are a developer and you want to update / enhance components used from the gridsuite commons-ui library
click [here](https://github.com/gridsuite/commons-ui) and follow instructions.

> This app uses @powsybl/network-viewer library which released in the npmjs packages.

Reviewer: ```suggestion
This app also uses [@powsybl/network-viewer](https://github.com/powsybl/powsybl-network-viewer) library which released on [npmjs](https://www.npmjs.com/package/@powsybl/network-viewer).
```
